packages/edit-site/src/components/sidebar-edit-mode/navigation-menu-sidebar/navigation-menu.js

```diff
@@ -2,12 +2,18 @@
  * WordPress dependencies
  */
 import {
-	__experimentalOffCanvasEditor as OffCanvasEditor,
+	__experimentalListView as ListView,
```

Comment:
This seems to be unused, so I guess this line can be deleted.

Reply:
How come the editor didn't catch this ...